work_with_shared_resources/using-shared-resource-csi-driver.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shared-resource-csi-driver"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: shared-resource-csi-driver
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource Container Storage Interface (CSI) Driver in any Kubernetes object that uses the volume type as `csi` to provision inline ephemeral volumes that contain the contents of Secret or ConfigMap objects. 
+{builds-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {ocp-product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: 
```

Comment:
{builds-title} isn't rendering correctly in the preview. Is there another variable you should use?

[builds for Red Hat OpenShift](https://85384--ocpdocs-pr.netlify.app/openshift-builds/latest/work_with_shared_resources/using-shared-resource-csi-driver.html) to provision...

Reply:
Actually, do you need the sentence, where you have similar wording in the previous sentence?